src/neural/cuda/common_kernels.cu

```diff
@@ -324,12 +324,12 @@ __global__ void expandPlanes_kernel_Fp16_NCHW(half* output,
 }
 
 void expandPlanes_Fp16_NCHW(half* output, const uint64_t* masks,
-                            const float* values, int n) {
+                            const float* values, int n, cudaStream_t stream = 0) {
```

Comment:
Consider removing the default value for stream here.
It seems a risky item to have a default on, and almost every single other function which takes a stream doesn't use a default.